langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -104,11 +190,30 @@ public isolated function filter(Type[] arr, @isolatedParam function(Type val) re
 # The combining function takes the combined value so far and a member of the array,
 # and returns a new combined value.
 #
-# For example
-# ```
-# reduce([1, 2, 3], function (int total, int n) returns int { return total + n; }, 0)
+# ```ballerina
+# int[] integers = [1, 2, 3];
+# int sum = integers.reduce(isolated function (int total, int next) returns int => total + next, 0);
+#
+#
```

Comment:
Better if this superfluous newline is removed.

Reply:
Please add this as a suggestion.